pysindy/pysindy.py

```diff
@@ -398,11 +400,20 @@ def print(self, lhs=None, precision=3):
             Precision to be used when printing out model coefficients.
         """
         eqns = self.equations(precision)
+        if isinstance(self.feature_library, SINDyPILibrary):
+            feature_names = self.get_feature_names()
+        else:
+            feature_names = self.feature_names
         for i, eqn in enumerate(eqns):
             if self.discrete_time:
-                print(self.feature_names[i] + "[k+1] = " + eqn)
-            elif lhs is None:
-                print(self.feature_names[i] + "' = " + eqn)
+                names = "(" + feature_names[i] + ")"
+                print(names + "[k+1] = " + eqn)
+            elif lhs is None and not isinstance(self.feature_library, SINDyPILibrary):
+                names = "(" + feature_names[i] + ")"
+                print(names + "' = " + eqn)
+            elif lhs is None and isinstance(self.feature_library, SINDyPILibrary):
+                names = feature_names[i]
+                print(names + " = " + eqn)
```

Comment:
This would be clearer with nested conditionals, e.g.
```python
elif lhs is None:
    if isinstance(self.feature_library, SINDyPILibrary):
        ...
    else:
        ...
```